src/client/pfs/pfs.proto

```diff
@@ -340,23 +291,27 @@ service InternalAPI {
 
   // Commit rpcs
   // StartCommit creates a new write commit from a parent commit.
-  rpc StartCommit(StartCommitRequest) returns (google.protobuf.Empty) {}
+  rpc StartCommit(StartCommitRequest) returns (Commit) {}
+  // Fork creates a commit on a new branch.
+  rpc Fork(ForkRequest) returns (Commit) {}
```

Comment:
Makes sense, I like `ForkCommit`. The other option I thought about was `BranchCommit` but I think it's a bad idea to use Branch as both a noun and a verb in our api.